package.json

```diff
@@ -5,7 +5,8 @@
   "main": "lib/index.js",
   "scripts": {
     "test": "mocha",
-    "test:coverage": "nyc npm test",
+    "test:jest": "jest --runInBand --config=jest.config.json",
```

Comment:
Plugin coverage tests for other plugins explode when executed in jest, hence it needs to be separated.